### PR TITLE
22 add an input helper

### PR DESCRIPTION
**Pull request by vazanoir**

### CODE REVIEW

File: src/components/register/RegisterForm.tsx [hunk 1]
```diff
@@ -44,24 +45,30 @@ const RegisterForm = () => {
...
+          onClick={mutate}
+          loading={isLoading}
+          disabled={!isValid(name)}
+        >
+          {buttonText()}
+        </Button>
+      </RowGroup>
+
+      <ErrorCard show={!isValidConditions.lteTwenty(name)}>
+        Maximum length is 20 chars
```
Comment (Brawdunoir) [inline src/components/register/RegisterForm.tsx:69]: What about i18n?

Comment (vazanoir) [reply src/components/register/RegisterForm.tsx:69]: i18n is a server side service

Comment (vazanoir) [reply src/components/register/RegisterForm.tsx:69]: and that's not the point of the pr

Comment (Brawdunoir) [reply src/components/register/RegisterForm.tsx:69]: i18n can and must be implemented on the client side, especially when displaying messages before querying anything to the api (how can it be a server side feature in this context ?) 
As such, it should be planned in advance in the front end app to limit future modifications.

It was just a quick note, and indeed it is not the goal of the PR.

**Review by Brawdunoir — APPROVED** [review]

**Pull request merged**